model/src/main/kotlin/HashAlgorithm.kt

```diff
@@ -128,9 +128,9 @@ enum class HashAlgorithm(private vararg val aliases: String, val verifiable: Boo
     /**
      * Return the hexadecimal digest of this hash for the given [resourceName].
      */
-    fun calculate(resourceName: String): String {
+    fun calculate(resourceName: String): String? {
```

Comment:
I am not sure about this change: As the function calculates a hash code over a resource - which is under control of ORT -, I would consider it a programming error if this resource cannot be resolved. In this sense, it would be fine if the function throws an exception - maybe not a NPE, but a `require` block could be added checking that `resource` is not null.

But I won't insist on that. Did the original NPE occur for unknown licenses or what was the reason for this change?

Reply:
This was not my change, but yes the NPE occured for unknown licenses, therefor I'm not sure if it should break the run for unknown licenses?

Reply:
Hm, then the mechanism to resolve known licenses seems to be fragile. Ok, then let's keep the code as it is now.